### PR TITLE
Make hints look more like Vimium

### PR DESCRIPTION
**Pull request by extratype**
* Use capital letters in labels.
* Ignore case when matching against the input, or, ignore Caps Lock.
* Change label style. It doesn't look as great as in Vimium, though.

### CODE REVIEW

File: src/HuntAndPeck/Views/OverlayView.xaml [hunk 1]
```diff
@@ -43,13 +43,13 @@
             <ItemsControl.ItemTemplate>
                 <DataTemplate>
                     <Grid>
-                        <Rectangle Width="{Binding Hint.BoundingRectangle.Width}" Height="{Binding Hint.BoundingRectangle.Height}" Stroke="Red" StrokeDashArray="1 2">
+                        <!--<Rectangle Width="{Binding Hint.BoundingRectangle.Width}" Height="{Binding Hint.BoundingRectangle.Height}" Stroke="Red" StrokeDashArray="1 2">
```
Comment (zsims) [inline src/HuntAndPeck/Views/OverlayView.xaml:46]: Can you please remove this?

**Review by zsims — CHANGES_REQUESTED** [review]

File: src/HuntAndPeck/Views/OverlayView.xaml (hunk 1 above)
Comment (zsims) [reply src/HuntAndPeck/Views/OverlayView.xaml:46]: There's a file conflict, normally I wouldn't worry about commented code :)

Comment (extratype) [reply src/HuntAndPeck/Views/OverlayView.xaml:46]: I resolved a conflict in `HintLabelService.cs`. There's no conflict now..

Comment (zsims) [reply src/HuntAndPeck/Views/OverlayView.xaml:46]: Thanks :+1:

**Comment by zsims** [comment]
Thanks for the awesome PRs @extratype, greatly appreciated

**Pull request merged**